### ISSUE TITLE
dchain - Decentralized Supply chain

### ISSUE THREAD

**Issue by nas214** [district0x/district-proposals#191]
dchain - Decentralized Supply chain

The last century has seen the rise of complex global supply chains. It started with a cross-border trading agreements that eased the coordination of multi-country supply chain, until we got to today's specialized manufacturing powerhouses where each of the most efficient geographical locations detain a certain part of a supply chain. Large enterprises use resource planning (ERP) to manage physical flows across these supply chains and geographical locations.
Over the last ten years, the increased digitization of producer/consumer interaction is allowing the supply chain manager to be more responsive to fluctuations in demand. This has been called the era of lean manufacturing, characterized by lower inventory and very complex (but more economical) coordination with multiple suppliers across the supply chain.
However, these supply chains remain linear in nature, and depend on central decision making/point-of-failure, and legacy based, but with more modern tools. 

In order to re-structure the most complex supply chains, and bring them to the decentralized marketplace era, there must be a platform on distric0x that will act as the hub (while being decentralized) for all other market places. This hub will act as a search engine, and be the starting point of any supply chain management. It will be fed the data incoming from the very complex and sometimes conflicting interactions of people, processes and objects across all the order chain, and will output a list of proposed  suppliers . In turn, this hub - having all the data (from clients & suppliers) and decision makers - will restructure the resources flow and re-direct them to the relevant marketplaces on distric0x. Firm boundaries between companies and suppliers become less important and coordination across the value chain becomes much more fluid.

dchain would be gathering data from a whole range of customers/suppliers and could, for example, effectively rout orders based on adjustable parameters (Volume, time-frame, minimum quality,...) through an optimized list of suppliers based on their past performance and reputation on their respective districts . 

Adjustments can be done in real time, while calibrating it to keep the parameters intact.
This will in turn create a network effect, as better coordination across the supply chain, and the use of reputation on district0x attract even better suppliers, leading to a more robust ecosystem.

0xBdf2019580DB3f51E54Bb96B813A2671B86eba7A

**Comment by imdying** [comment]
I like the idea.

Can you give an example of how an existing business would use it and what
advantages it would give them over their competitors?

Supply chain / logistics software is really complex. Don't think this is
too big a task for district0x?

Also what is the size of the target market for supply chain solutions?

**Comment by nas214** [comment]
It would be like an Amazon on Steroids.
In order to have a working supply chain you need many actors/suppliers... Instead of having to find them one by one on the district, dchain would be the aggregator of all those actors.

**Comment by imdying** [comment]
why would I use the district and not just sell through amazon?

**Comment by nas214** [comment]
1-Amazon is not capable of building a supply chain, it is only possible to have simple exchanges of goods and money (This is why I refereed to it as Amazon on Steroids). What dchain actually is, is more like a virtual space governed by a type of decentralized Enterprise Resources planning module (ERP)  that shares the same reputation, search and invoicing systems of potential suppliers on districts in district0x. It would get the information needed in real time and translate it to orders to districts based on the parameters set by the issuers.
2-Amazon is centralized.

**Comment by MCDForm** [comment]
At this point Amazon does everything except manufacture.  I'm not sure if Amazon is purely centralized.  So much of the marketplace never touches Amazons hands.  The product is sold on Amazon but when it's shipped from outside the US it doesn't have to even touch Amazon at all.  Boxes enter the US with Fedex and UPS stickers already attached.  Cargo moves straight from the port to shipping hubs where products is shipped straight to the owner.  Many sellers on Amazon live in one country, buy from manufacturers in another and then ship to the US without ever leaving their home.

But like I said they haven't touch manufacturing yet.

**Comment by nas214** [comment]
I agree. dchain does not manufacture anything too. It's the access gateway to the manufacturing "hubs", for products which need a long and complex supply chain that cannot be solved by B2B solutions. 
On a side note all the current manufacturing model is very likely to be disrupted by the advent of 3D printing which will level the playing field between the manufacturing capabilities/costs of different Geographies.
